backend/src/Controllers/getElectionResultsController.ts

```diff
@@ -48,8 +49,11 @@ const getElectionResults = async (req: IElectionRequest, res: Response, next: Ne
         }
         const msg = `Tabulating results for ${voting_method} election`
         Logger.info(req, msg);
-        results[race_index] = VotingMethods[voting_method](candidateNames, cvr, num_winners)
+        let rng = seedrandom(election.election_id + ballots.length.toString())
+        const tieBreakOrders = election.races[race_index].candidates.map((Candidate) => (rng() as number))
```

Comment:
I was initially confused here since rng() outputs floats, but the test cases work with lists of unique integers. But it looks like it'll work fine. I guess it's technically possible for rng() to output the same number twice? but I'm not worried about that

Reply:
I'm not worried about duplicate floats. And I think in JS integers are just treated as floats.